pysumma/preprocessing/create_initial_condition.py

```diff
@@ -0,0 +1,152 @@
+from osgeo import gdal
```

Comment:
Minor comment, but I think a better name would just be `initial_condition.py` rather than `create_initial_condition.py` so that we can have it be more general than just methods for making the initial conditions, but also for modifying them.

Reply:
Same goes for the `create_local_attribute` and `create_param_trial`